### PR TITLE
fix: inline error for username taken (guest post/thread), refs #841

### PR DESCRIPTION
**Pull request by Nik101010**
#841

### CODE REVIEW

**Comment by Sama34** [comment]
@Nik101010

**Comment by Nik101010** [comment]
@Sama34 sanatizing usernames in the build_postbit function would make sense, but this could lead to stuff beeing double escaped... 
I reviewed the whole files, I found one path were the username was not sanatized... added it in this case

**Comment by PaulBender** [comment]
Everything looks good to me, if no one else has any issues, this can be merged.

File: newreply.php [hunk 1]
```diff
@@ -1074,7 +1055,7 @@
 			$post = $db->fetch_array($query);
 			if(!$mybb->user['uid'] || !$post['username'])
 			{
-				$post['username'] = $mybb->get_input('username');
+				$post['username'] = htmlspecialchars_uni($mybb->get_input('username'));
```
Comment (Sama34) [inline newreply.php]: Please revert this. the `build_postbit` function will handle this on another PR.

**Comment by Nik101010** [comment]
@Sama34 I reverted the last commit ...

**Pull request merged**